src/FSharp.Core/prim-types-prelude.fs

```diff
@@ -6,6 +6,11 @@ namespace Microsoft.FSharp.Core
     // Basic type abbreviations
 
     type obj = System.Object
+#if BUILDING_WITH_LKG || NO_NULLCHECKING_LIB_SUPPORT
```

Comment:
What is building with LKG --- shouldn't NO_NULLCHECKING_LIB_SUPPORT be sufficient?

Reply:
**L**ast **K**nown **G**ood compiler.
This one is used when building the plain FSharp.Compiler.Service solution **without** running the build script.